### DISCUSSION TITLE
Is it possible to have a single toast and update it automatically when calling toast methods?

### DISCUSSION THREAD

**Discussion by phiter** [fkhadra/react-toastify#823] [Q&A]
In my application there is no need to have multiple toast notifications. I only need one toast to show up.

I know I can update toasts by using `toast.update`, but is it possible to make any call to `toast` or `toast.success`, etc to update the current active toast? Passing the same ID just prevents duplicates instead of updating the toast. So, is it possible?

**Comment by ChrisBoon** [comment]
Old question, but answering in case anyone else has the same issue.
You can create this behavior is userland using the toast.isActive(id).
Simplified example: 
```
const addOrUpdateToast = (props, content)=> {
  
  if(props.id && toast.isActive(props.id) ){
    const {id, ...rest } = props;
    toast.update(id, { render: content, ...rest });
  } else {
    toast(content, props);
  }    
}
```

The only thing I'm not sure about is how you'd handle queued toasts if the container has a limit. It's not something I've used so not had to explore.
